sdk/storage/Azure.Storage.DataMovement/src/UriToStreamJobPart.cs

```diff
@@ -292,6 +292,19 @@ internal async Task LengthKnownDownloadInternal()
                         await CompleteFileDownload().ConfigureAwait(false))
                         .ConfigureAwait(false);
                 }
+
+                // Set rangeSize
+                long rangeSize = CalculateBlockSize(totalLength);
+
+                // Get list of ranges of the blob
+                IList<HttpRange> ranges = GetRangesList(0, totalLength, rangeSize);
+
+                // Create Download Chunk event handler to manage when the ranges finish downloading
+                _downloadChunkHandler = GetDownloadChunkHandler(
+                    currentTranferred: 0,
+                    expectedLength: totalLength,
+                    ranges: ranges,
+                    jobPart: this);
```

Comment:
So this will essentially try a paritioned download if our single download failed for some reason? Is that something we actually want to do? Any issues that could produce unexpectged results here?

Reply:
Ah you're right.

I think this was a mistake merge. I was trying to fix something with the download but that was before you already fixed it. I can remove this. Looks like this didn't really do anything since the download would complete anyways.